d2l-input-textarea.js

```diff
@@ -135,6 +135,11 @@ $_documentContainer.innerHTML = `<dom-module id="d2l-input-textarea">
 				@apply --d2l-input-invalid;
 			}
 
+			:host([no-padding]) textarea.d2l-input {
+				padding-left: 0px;
```

Comment:
Nit: our general practice is to omit units on 0 values.  Once upon a time, I think it was a micro-optimization.